Pod/Classes/VOKCoreDataManager.h

```diff
@@ -25,10 +25,14 @@ typedef NS_ENUM (NSInteger, VOKMigrationFailureOption) {
     VOKMigrationFailureOptionNone,
     /// Silently delete and recreate the database file, data will be erased, but instability and crashing will be avoided
     VOKMigrationFailureOptionWipeRecovery,
-    /// Same as VOKMigrationFailureOptionWipeRecoveryAndAlert, but will also notify the user that data has been erased via an on screen alert. NOTE: not yet implemented on OS X
-    VOKMigrationFailureOptionWipeRecoveryAndAlert,
 };
 
+/**
+ * Name of the NSNotification that's posted after a successful DB wipe following a migration failure.
```

Comment:
If you're adding this notification on success, worth adding one for failure too?